code/datums/mood_events/drug_events.dm

```diff
@@ -37,3 +37,16 @@
 
 /datum/mood_event/withdrawal_critical/add_effects(drug_name)
 	description = "<span class='boldwarning'>[drug_name]! [drug_name]! [drug_name]!</span>\n"
+
+/datum/mood_event/happiness_drug
+	description = "<span class='nicegreen'>I can't feel anything and I never want this to end.</span>\n"
+	mood_change = 50
+
+/datum/mood_event/happiness_drug_good_od
+	description = "<span class='nicegreen'>YES! YES!! YES!!!</span>\n"
+	mood_change = 100

```

Comment:
The bad overdose moodlet times out; why doesn't the good overdose moodlet?

Reply:
oversight, I'll fix it